Formula/quantlib.rb

```diff
@@ -38,12 +37,20 @@ class Quantlib < Formula
   def install
     ENV.cxx11 if build.cxx11?
     if build.head?
-      Dir.chdir "QuantLib"
       system "./autogen.sh"
     end
+    if MacOS.version >= :mavericks
+      ENV.append "LDFLAGS", "-stdlib=libstdc++ -mmacosx-version-min=10.6"
+      ENV.append "CXXFLAGS", "-stdlib=libstdc++ -mmacosx-version-min=10.6"
+    end
     system "./configure", "--disable-dependency-tracking",
+                          "--enable-static",
+                          "--with-boost-include=/usr/local/include/",
+                          "--with-boost-lib=/usr/local/lib/",
```

Comment:
These paths cannot be hard coded.  You can use `Formula["boost"].opt_lib` and `.opt_include`